tests/scripts/thread-cert/border_router/test_advertising_proxy.py

```diff
@@ -140,7 +140,7 @@ def test(self):
         #
 
         client.srp_client_set_host_address('2001::2')
-        self.simulator.go(2)
+        self.simulator.go(4)
```

Comment:
```suggestion
        self.simulator.go(10)
```

nit: We can use longer time for stability.